io.github.agam778.MS-Office-Electron.yml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.agam778.MS-Office-Electron
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: start-ms-office-electron
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    npm_config_nodedir: /run/build/ms-office-electron/flatpak-node/cache/node-gyp/18.2.4/
+modules:
+  - name: ms-office-electron
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/ms-office-electron/flatpak-node/cache
+    sources:
+      - generated-sources.json
+      - type: git
+        url: https://github.com/agam778/MS-Office-Electron/
+        tag: v0.7.2
+        dest: main
+      - type: file
+        url: https://github.com/agam778/MS-Office-Electron/raw/main/assets/generated/icons/png/512x512.png
+        sha256: 1a850da5e62f748544dda46f72c756bb63e740ee4ddc7f0f97833f963f9469f3
+      - type: file
+        url: https://raw.githubusercontent.com/agam778/MS-Office-Electron/main/assets/flatpak/io.github.agam778.MS-Office-Electron.desktop
+        sha256: 13134aa60bd1465fd438c0844f0a1b8d38401aedf05f70d8eda56e8d5650b4ed
+      - type: file
+        url: https://raw.githubusercontent.com/agam778/MS-Office-Electron/main/assets/flatpak/io.github.agam778.MS-Office-Electron.metainfo.xml
+        sha256: d13ecd355ff45ea4c343dd8e7142bc898b46fc766558739218bf21d4681c5500
+      - type: script
+        dest-filename: start-ms-office-electron.sh
+        commands:
+          - zypak-wrapper.sh /app/ms-office-electron/ms-office-electron "$@"
+    build-commands:
+      - mv 512x512.png io.github.agam778.MS-Office-Electron.png
+      - HOME=$PWD yarn config --offline set yarn-offline-mirror $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror
```

Comment:
Such hack is unnecessary. Just add a source with proper `.yarnrc`, e.g.
```yaml
sources:
  ...
  - type: inline
    contents: |
      yarn-offline-mirror /run/build/ms-office-electron/flatpak-node/yarn-mirror
    dest: main
    dest-filename: .yarnrc
```
You can also add `--install.offline true` and `--run.offline true` to the `.yarnrc`, then you won't need to pass `--offline` arg yarn commands below.

Reply:
Done, please check.